### PR TITLE
Added "Blocked" Profile service verification page

### PR DESCRIPTION
**Pull request by shubhamsigdar1**
### Issue Closes https://github.com/Real-Dev-Squad/website-www/issues/732

Parent Issue Ticket: https://github.com/Real-Dev-Squad/website-www/issues/730

**Design Doc:** https://docs.google.com/document/d/1ra6Nagmfbywhc8-_fRtmv1K688uDwqLQTOH-Om2t7oE/edit?usp=sharing

### What is the change?

- added status(PENDING, VERIFIED, BLOCKED) into constant file
- render h3 tag, description and button 
- When user click on button then it redirect to chaincode page

### Is Development Tested?

- [X] Yes
- [ ] No

### Is under feature flag?

- [X] Yes
- [ ] No

### Frontend Changes

- [X] Yes
- [ ] No

### Test stats

https://github.com/Real-Dev-Squad/website-www/assets/107163260/5d2f7adc-1921-4394-9398-2839a2ac35a4

### Before / After Change Screenshots

After :

https://github.com/Real-Dev-Squad/website-www/assets/107163260/24dbe5eb-68e8-4376-9f08-52a746dc8ad5

### CODE REVIEW

**Review by ajoykumardas12 — APPROVED** [review]

File: app/components/identity-steps/step-seven.hbs [hunk 1]
```diff
@@ -1,36 +1,60 @@
...
+      >
+        {{#if (eq currentStatus.status 'PENDING')}}
+          <p data-test-verification-description1>
+            Refresh to Check Verification Status
+          </p>
+          <p data-test-verification-description2>
+            Your Profile Service Linked with Real Dev Squad Service
+          </p>
+        {{else if (eq currentStatus.status 'BLOCKED')}}
+          <p data-test-verification-description1>
```
Comment (ajoykumardas12) [inline app/components/identity-steps/step-seven.hbs:32]: The data-test attributes can be improved. 
If using number suffix, `description-1` might be better for readability.
Also, the container div can be `description-container`

Comment (shubhamsigdar1) [reply app/components/identity-steps/step-seven.hbs]: Done with changes

**Review by ajoykumardas12 — APPROVED** [review]

File: app/components/identity-steps/step-seven.js [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+  currentProfileStatus = [
+    {
+      status: PROFILE_STATUS.PENDING,
+      heading: 'Pending',
+      icon: 'hourglass-half',
+    },
+    {
+      status: PROFILE_STATUS.BLOCKED,
+      heading: 'Blocked',
+      icon: 'ban',
```
Comment (sahsisunny) [inline app/components/identity-steps/step-seven.js:18]:
```suggestion
      icon: 'blocked',
```

**Review by sahsisunny — APPROVED** [review]

File: app/components/identity-steps/step-seven.js (hunk 2 above)
Comment (shubhamsigdar1) [reply app/components/identity-steps/step-seven.js:18]: It is icon name from font-awesome

**Pull request merged**